clients/apiextensions/assets.go

```diff
@@ -1,43 +1,21 @@
 package apiextensions
 
 import (
-	"strconv"
-
 	"github.com/iotaledger/wasp/clients/apiclient"
 	"github.com/iotaledger/wasp/packages/coin"
 	"github.com/iotaledger/wasp/packages/isc"
 )
 
-// TODO: Handle Coins (other than base tokens) and Objects
+// TODO: Handle Objects
```

Comment:
Might make sense to just have one balance route, which returns everything an account has. Coins and Objects. Then I'd make sense to add objects here

Reply:
Should I rename this func for Balance ( Coins) only?